src/checkstyle/checkstyle.xml

```diff
@@ -179,10 +182,6 @@
         <module name="IllegalThrows">
             <property name="illegalClassNames" value="java.lang.Error, java.lang.Throwable"/>
         </module>
-        <module name="RedundantThrows">
-            <property name="allowUnchecked" value="true"/>
-            <property name="allowSubclasses" value="true"/>
-        </module>
```

Comment:
This was removed in a version of `checkstyle` between the previous version and the current one, without replacement (as far as I could find.)